webapp/certification/views.py

```diff
@@ -15,6 +15,99 @@
 )
 
 
+def certification_hardware_details(canonical_id, release):
+    models = api.certified_models(canonical_id=canonical_id)["objects"]
```

Comment:
You could do:
```
try:
    model = api.certified_models(canonical_id=canonical_id)["objects"][0]
except KeyError:
    abort(404)
```
and then you would can just replace occurences of `models[0]` with `model`

Reply:
Done